docs/installation/production/manual/configuration/advanced/multi_tenancy.rst

```diff
@@ -0,0 +1,137 @@
+.. _multi_tenancy_config:
```

Comment:
Rebase this branch with main and add Tenants to the What's New feature list for 4.4 with a link to this doc.